apps/kyb-app/src/pages/CollectionFlow/versions/v2/hooks/useCollectionFlowContext/helpers/get-document-ids-from-context.ts

```diff
@@ -0,0 +1,62 @@
+import { UIPage, UISchema } from '@/domains/collection-flow';
+import { CollectionFlowContext } from '@/domains/collection-flow/types/flow-context.types';
+import { TDocument } from '@ballerine/common';
+import {
+  formatValueDestination,
+  getFieldDefinitionsFromSchema,
+  getFileOrFileIdFromDocumentsList,
+  IFormElement,
+  isDocumentFieldDefinition,
+  TBaseFields,
+  TDeepthLevelStack,
+} from '@ballerine/ui';
+import get from 'lodash/get';
+
+export const getDocumentIdsFromContext = (context: CollectionFlowContext, uiSchema: UISchema) => {
+  const documentIds: string[] = [];
+
+  const run = (elements: Array<IFormElement<TBaseFields, any>>, stack: TDeepthLevelStack = []) => {
+    for (const element of elements) {
+      if (isDocumentFieldDefinition(element)) {
+        const documents = get(context, formatValueDestination(element.valueDestination, stack));
+        const document = documents?.find(
+          (doc: TDocument) => doc.id === element.params?.template?.id,
+        );
+
+        if (!document) {
+          continue;
+        }
+
+        const documentId =
```

Comment:
Documents and document files are separate entities. Document file has an id of its own. The logic here it's either wrong or it's a naming issue.